src/earn/EarnHome.tsx

```diff
@@ -247,7 +260,21 @@ export default function EarnHome({ navigation, route }: Props) {
     learnMoreBottomSheetRef.current?.snapToIndex(0)
   }
 
-  const zeroPoolsinMyPoolsTab = displayPools.length === 0 && activeTab === EarnTabType.MyPools
+  const onPressTryAgain = () => {
+    AppAnalytics.track(EarnEvents.earn_home_error_try_again)
+    dispatch(refreshPositions())
+  }
+
+  const positionsStatus = useSelector(positionsStatusSelector)
+  const positionsFetchedAt = useSelector(positionsFetchedAtSelector)
+  const errorLoadingPools =
+    positionsStatus === 'error' &&
+    (pools.length === 0 ||
+      (!!positionsFetchedAt &&
```

Comment:
Do we need `!!positionsFetchedAt` here, if positionsFetched at returns `0` it looks like we will attempt to display expired pools.

Reply:
without it I get a typescript error because `positionsFetchedAt` can be undefined, so I think I need to check that it is not undefined before doing the math, is there a way I should do that other than `!!`?